builder_test.go

```diff
@@ -168,6 +169,86 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 				}
 			})
 
+			it("copies any created BOM files to the correct locations", func() {
+				bpA := testmock.NewMockBuildpack(mockCtrl)
+				bpB := testmock.NewMockBuildpack(mockCtrl)
+				buildpackStore.EXPECT().Lookup("A", "v1").Return(bpA, nil)
+				buildpackStore.EXPECT().Lookup("B", "v2").Return(bpB, nil)
+
+				bomFilePath1 := filepath.Join(layersDir, "launch.bom.cdx.json")
```

Comment:
Should we also test that build and cache bomfiles are copied?

Reply:
_Fixed in latest commit._